src/Storages/ObjectStorageQueue/ObjectStorageQueueMetadata.cpp

```diff
@@ -654,12 +654,26 @@ size_t ObjectStorageQueueMetadata::unregisterActive(const StorageID & storage_id
     const auto registry_path = zookeeper_path / "registry";
     const auto table_path = registry_path / getProcessorID(storage_id);
 
-    zk_client->tryRemove(table_path);
+    auto code = zk_client->tryRemove(table_path);
     const size_t remaining_nodes_num = zk_client->getChildren(registry_path).size();
 
-    LOG_TRACE(
-        log, "Removed {} from active registry (remaining: {})",
-        storage_id.getFullTableName(), remaining_nodes_num);
+    if (code == Coordination::Error::ZOK)
+    {
+        LOG_TRACE(
+            log,
+            "Table '{}' has been removed from the active registry (remaining nodes: {})",
+            storage_id.getFullTableName(),
```

Comment:
LGTM. My only nitpick is that in this method we're reporting the FQN of the table, while in the rest we're using `self.table_id` which uses the UUID if present and falls back to the FQN.

For consistency, I think it'd be better to do at the top of this method the usual:

```cpp
const auto self = Info::create(storage_id);
```